python/cuml/comm/serialize.py

```diff
@@ -25,11 +27,34 @@
 serializable_classes = (cuml.common.CumlArray,)
 
 
+def serialize_mat_descriptor(m):
+    return cp.cusparse.MatDescriptor.create, ()
+
+
 try:
     from distributed.protocol import dask_deserialize, dask_serialize
     from distributed.protocol.cuda import cuda_deserialize, cuda_serialize
     from distributed.utils import log_errors
 
+    from distributed.protocol import register_generic
+
+    from cuml.naive_bayes.naive_bayes import MultinomialNB
+
+    try:
+
+        # Requires Dask Distributed 2.12.0+
+
+        register_generic(MultinomialNB, "cuda",
+                         cuda_serialize, cuda_deserialize)
+        register_generic(MultinomialNB, "dask",
+                         dask_serialize, dask_deserialize)
+    except TypeError:
+        pass
+
+    register_generic(MultinomialNB)
```

Comment:
So this is the same as `register_generic(MultinomialNB, "dask", dask_serialize, dask_deserialize)`. If we prefer to keep that line above, maybe we can move this into the `except TypeError` block?

```suggestion
        register_generic(MultinomialNB)
```